.github/workflows/pluto-workflow-kustomize.yaml

```
@@ -49,7 +54,10 @@ jobs:
 
           ## Helm
           else
-            helm template "charts/${{ inputs.service_name }}" --values "${{ inputs.values_base }}" --values "${{ inputs.values_overlay }}" > "$MANIFEST_FILE"
+            # if an user wishes to customize a Helm chart to use
+            CHART_NAME=$([ -n "${{ inputs.chart_name }}" ] && echo "${{ inputs.chart_name }}" || echo "${{ inputs.service_name }}")
```

- Since we're using `bash`, this can be simplified using default variable substitution.
- Quoting.

```suggestion
            CHART_NAME="${{ inputs.chart_name }}"
            CHART_NAME="${CHART_NAME:-${{ inputs.service_name }}}"
```